tools/nl/embeddings/build_embeddings.py

```diff
@@ -71,13 +89,22 @@ def main(_):
     utils.save_embeddings_lancedb(fm.local_output_dir(), final_embeddings)
   else:
     raise ValueError(f'Unknown store type: {index_config.store_type}')
+  logging.info("Saved embeddings.")
 
   # Save index config
   utils.save_index_config(fm, index_config)
+  logging.info("Saved index config.")
+
+  # Save custom catalog
+  if custom_catalog:
+    utils.save_custom_catalog(fm, custom_catalog)
+    logging.info("Saved custom catalog.")
```

Comment:
I don't think this tools should save the catalog, this is the input?

Reply:
For the server to be able to load at startup, it needs to be persisted somewhere. Do you have an alternative in mind?